src/ui/pages/SidePage/components/IncomingRequest/components/SignTransactionRequest.tsx

```diff
@@ -74,29 +66,34 @@ const SignTransactionRequest = ({
           alt={requestData.label}
         />
         <h2 className="sign-name">{requestData.label}</h2>
-        <h3 className="sign-subtitle">{signTransaction?.eventName}</h3>
-        <p className="sign-link">{signTransaction?.ownerUrl}</p>
+        <h3 className="sign-subtitle">{signTransaction?.type}</h3>
```

Comment:
The type is an enum we have - this can't be a user facing string. Let me ask Robert what we put here

Reply:
OK this is being removed - so you can remove this line and remove the class if its not used anymore